share/fathead/git_manual/output.txt

```diff
@@ -0,0 +1,413 @@
+git-config	A										Get and set repository or global options
+<pre><code>git config [<file-option>] [type] [-z|--null] name [value [value_regex]]
+git config [<file-option>] [type] --add name value
+git config [<file-option>] [type] --replace-all name value [value_regex]
+git config [<file-option>] [type] [-z|--null] --get name [value_regex]
+git config [<file-option>] [type] [-z|--null] --get-all name [value_regex]
+git config [<file-option>] [type] [-z|--null] [--name-only] --get-regexp name_regex [value_regex]
+git config [<file-option>] [type] [-z|--null] --get-urlmatch name URL
+git config [<file-option>] --unset name [value_regex]
+git config [<file-option>] --unset-all name [value_regex]
+git config [<file-option>] --rename-section old_name new_name
+git config [<file-option>] --remove-section name
+git config [<file-option>] [-z|--null] [--name-only] -l | --list
+git config [<file-option>] --get-color name [default]
+git config [<file-option>] --get-colorbool name [stdout-is-tty]
+git config [<file-option>] -e | --edit</pre></code>	https://git-scm.com/docs//config.html
```

Comment:
The abstract will have to go on a single line.  You can replace the newlines with a `<br>` tag to format it for the front end.